### ISSUE TITLE
Manage strike with HTML Text styling.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#136]
```
Manage strike with HTML Text styling for docx and odt by using the <strike> or 
<s> HTML tag.
```

Original issue reported on code.google.com by `angelo.z...@gmail.com` on 30 Aug 2012 at 12:03

**Comment by GoogleCodeExporter** [comment]
```
strike feature is now available for 1.0.0
```

Original comment by `angelo.z...@gmail.com` on 30 Aug 2012 at 12:45

**Comment by GoogleCodeExporter** [comment]
```
strike feature is now available for 1.0.0
```

Original comment by `angelo.z...@gmail.com` on 30 Aug 2012 at 12:46
- Changed state: **Fixed**
